db/seeds.rb

```diff
@@ -21,6 +22,9 @@
   end
 end
 
+DatabaseCleaner.strategy = :truncation, {:only => %w[faqs]}
```

Comment:
We can use this DatabaseCleaner gem to give other tables a clean slate in seeds.rb as we deem necessary and useful.